e2e/libs/recurring_job/crd.py

```diff
@@ -10,25 +9,25 @@ def __init__(self):
         self.rest = Rest()
 
     def create(self, name, task, groups, cron, retain, concurrency, label):
-        logging.warn("NotImplemented")
+        logging("Delegating the create call to API because there is no CRD implementation")
         return self.rest.create(name, task, groups, cron, retain, concurrency, label)
 
     def delete(self, job_name, volume_name):
-        logging.warn("NotImplemented")
+        logging("Delegating the delete call to API because there is no CRD implementation")
         return self.rest.delete(job_name, volume_name)
 
     def get(self, name):
-        logging.warn("NotImplemented")
+        logging("Delegating the get call to API because there is no CRD implementation")
```

Comment:
For getter function, don't suggest having logs to prevent log flooding.

Reply:
Removed.